src/components/pages/GlobalSettings/General/YourInstallation/YourInstallation.tsx

```diff
@@ -0,0 +1,35 @@
+import {FullWidthSpace} from '@custom-antd';
+
+import {ConfigurationCard} from '@molecules';
+
+import {useApiDetailsField} from '@src/store/apiDetails';
+
+import env from '../../../../../env';
+
+import {YourInstallationContainer} from './YourInstallation.styled';
+import YourInstallationItem from './YourInstallationItem';
+
+const YourInstallation: React.FC = () => {
+  const [data] = useApiDetailsField('data');
+
+  const versionRegex = /^(\d+\.)?(\d+\.)?(\*|\d+)$/;
```

Comment:
* May be worth to allow`v` on the beginning (`v?`)
* We support tagged (like alpha, beta) and dirty versions (basically - instead of `$`, `([.-].+)?$` would be fine
* Why do we allow `*`, `1.*` and `1.*.*` as a version?

Reply:
1. Api sending without it, I don't think so
2. this is not working. Modified so it supports dirty versions
3. Updated

Reply:
Regarding (2) - what is not working? Besides, should we actually check the version at all, instead of showing it always when we receive it?

Reply:
In case of api version, sometimes it is sending a commit, in that case we want to display # before it. We might not check for other fields

Reply:
Why we wouldn't want to show the commit? We tag the images with commit i.e. for DEV version. Besides, you may as well use SHA hash for specific Docker images too.

Reply:
I don't understand the problem here. I just display the version, in case it is not formatred like version, I display it with #, because it is a commit